www/source/partials/docs/_dev-pkgs-share.html.md.erb

```diff
@@ -31,23 +31,23 @@ When you upload a package to Builder, you are required to supply an auth token a
 
 Once you have this token, you can set the `HAB_AUTH_TOKEN` [environment variable](/docs/reference#environment-variables) to this value, so that any commands requiring authentication will use it.
 
-### Create origin keys
+### Create an origin key pair
 
-After you have done the basic account creation steps, you need to create your origin keys. The private key will be used to sign your packages and the public key will be used by supervisors to verify the integrity of your packages (`.hart` files).
+After finishing the basic account creation steps, you need to create your origin key pair. You will use the private key to sign your packages and Supervisors will use the public key to verify the integrity of your packages (`.hart` files).
```

Comment:
`Supervisors` don't do the verification; it's just a part of the `hab` commands that install or verify artifacts.

Reply:
Do `Supervisors` at least invoke the verification? Who/what/where is the verification done from?

Reply:
There are instances where supervisor code triggers an install: either if you `hab svc load` a heretofore uninstalled package or if automatic updates are enabled. However, in both of these cases, the install (and thus verification) procedure is using the exact same code as if the user had done a `hab pkg install`, which doesn't require a supervisor be running at all.